week-8/Homework/mandatory/2-fetch-exercise/index.html

```diff
@@ -1,28 +1,32 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>CodeYourFuture - Fetch</title>
-  <meta name="description" content="Introduction to Fetch">
-  <meta name="author" content="CodeYourFuture">
-  <style>
-  #greeting-text {
-    font-size: 20px;
-    border: 1px solid #ccc;
-    background-color: #ddd;
-    width: 20%;
-    text-align: center;
-    margin: 0 50px;
-    padding: 30px;
-  }
-  </style>
-</head>
-
-<body>
-  <h1>A greeting in a random language...</h1>
-  <p id="greeting-text"></p>
-
-  <script src="exercise.js"></script>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <title>CodeYourFuture - Fetch</title>
+    <meta name="description" content="Introduction to Fetch" />
+    <meta name="author" content="CodeYourFuture" />
+  </head>
+  <body>
+    <h1>A greeting in a random language...</h1>
+    <p id="greeting-text"></p>
+    <style>
+      body {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        margin-top: 100px;
+      }
+      #greeting-text {
+        font-size: 20px;
+        border: 1px solid #ccc;
+        background-color: #ddd;
+        width: 20%;
+        text-align: center;
+        margin: 0 50px;
+        padding: 30px;
+      }
+    </style>
+    <script src="exercise.js"></script>
```

Comment:
Internal style is not a good idea most of the time, unless for small files like this. But is there a reason to use it inside the body ?
To give more information, styling was not allowed to be in anywhere except the header till recently, with HTML5 and above. It could be a good practice to give it the old organized style

Reply:
Separated all files.